source/overview/security.rst

```diff
@@ -79,40 +138,45 @@ Security updates address newly discovered attacks reported to Mattermost, Inc. b
 For more information, please see:
 
 - `Mattermost Security Updates Disclosures <http://about.mattermost.com/security-updates/>`_
-   - A summary of security updates made based on past and on-going security analysis and penetration testing.
+A summary of security updates made based on past and on-going security analysis and penetration testing.
 
 - `Mattermost Responsible Disclosure Policy <https://www.mattermost.org/responsible-disclosure-policy/>`_
-   - An overview of how security issues are confidentially reported to and address by Mattermost, Inc.
+An overview of how security issues are confidentially reported to and address by Mattermost, Inc.
 
 Security Policies
 ------------------------------------
 
-For information on internal security policies, development guidelines, business continuity plans and common security-related questions from enterprises, please see our `Security Policies <https://docs.mattermost.com/process/security.html>`_ documentation.
+For information on internal security policies, 
+development guidelines, 
+business continuity plans,
+and common security-related questions from enterprises, 
+please see our `Security Policies <https://docs.mattermost.com/process/security.html>`_ documentation.
 
 
 HIPAA compliance
 ------------------------------------
 
 Deploying Mattermost as part of a HIPAA-compliant IT infrastructure requires a deployment team trained on `HIPAA-compliance requirements and standards <http://www.hhs.gov/hipaa/for-professionals/security/laws-regulations/>`_.
 
-- Mattermost offers HIPAA-relevant **Technincal Safeguards** including:
+Mattermost offers HIPAA-relevant *Technincal Safeguards* including:
 
-     - `Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
-     - `Access Control <https://docs.mattermost.com/overview/security.html#access-control-policy>`_
-     - `Transmission Security <https://docs.mattermost.com/overview/security.html#transmission-security>`_
-     - `Audit Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
+- Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
```

Comment:
Need the ` added to the beginning to format these as links

Reply:
done